js/core/dfn.js

```diff
@@ -21,11 +21,13 @@ define(
                     } else {
                         dfn.attr("data-dfn-for", (dfn.closest("[data-dfn-for]").attr("data-dfn-for") || "").toLowerCase());
                     }
-                    var title = dfn.dfnTitle();
-                    if (!conf.definitionMap[title]) {
-                        conf.definitionMap[title] = [];
-                    }
-                    conf.definitionMap[title].push(dfn);
+                    var titles = dfn.dfnTitle( true );
+                    $.each(titles, function() {
```

Comment:
If possible, please avoid using jQuery for loops. We are trying to systematically remove this dependency where not needed.